### ISSUE TITLE
Clarity Stack View is missing visual focus indication

### ISSUE THREAD

**Issue by bashibozuk** [vmware/clarity#4899]
## Describe the accessibility issue

Stack View do not have a clear visual indication of focus. For Stack Views that are collapsed, visual focus is indicated using a slight background color change. For Stack View that are expanded, no visual focus indicator is available.

## WCAG Guideline

_What WCAG guideline is being violated here and how? You should reference a guideline from https://www.w3.org/TR/WCAG21/_
https://www.w3.org/TR/WCAG21/#focus-visible

## How to reproduce

https://stackblitz.com/edit/clarity-v3-dark-theme-smp8kz?file=src/app/app.component.html

You can see here that the focused state of the stack view is not the recommended outline in WCAG, but slight color change 

1.  Open https://stackblitz.com/edit/clarity-v3-dark-theme-smp8kz?file=src/app/app.component.html
2.  Press Tab key until the stack view is focused
3. Observe that the no outline is present to mark that a keyboard interaction enabled element is focused
![stack-view-focus-indication](https://user-images.githubusercontent.com/2686283/90115091-4904c300-dd5c-11ea-8f73-52463d46afb0.png)

## Expected behavior

When a clickable header of a StackView is focused a visual focus indication should be provided
![accordion-focus-indication](https://user-images.githubusercontent.com/2686283/90115124-5752df00-dd5c-11ea-96ca-e2e59ade0972.png)

## Versions

**App**

- Angular: 9
- Node: 12
- Clarity: 3

**Device:**

- Type:  MacBook
- OS: macOs Catalina
- Browser chrome
- Version 
- Screen Reader: N/A
